ci/pipelines/pipeline.yml

```diff
@@ -0,0 +1,483 @@
+  type: git-tag
+  icon: tag
+  source:
+    uri: https://github.com/RoboJackets/apiary-mobile
+    username: x-access-token
+    password: (("github.com"/token.token))
+    tag_filter: v*
+  tags:
+    - resources
+
+- name: tests-check
+  type: github-check
+  icon: check
+  public: true
+  source:
+    check_name: Apiary Android Tests
+    resource_name: tests-check
+  tags:
+    - resources
+
+- name: pr-build-check
+  type: github-check
+  icon: check
+  public: true
+  source:
+    check_name: Pull Request Build
+    resource_name: pr-build-check
+  tags:
+    - resources
+
+- name: internal-test-upload-check
+  type: github-check
+  icon: check
+  public: true
+  source:
+    check_name: Internal Test Draft Release
+    resource_name: internal-test-upload-check
+  tags:
+    - resources
+
+- icon: webhook
+  name: webhooks
+  public: true
+  source: null
+  tags:
+  - resources
+  type: github-webhook
+
+- icon: slack
+  name: slack
+  public: true
+  source: null
+  tags:
+  - resources
+  type: slack
+  
+
+jobs:
+  - name: build-release
+    on_abort:
+      inputs: []
+      params:
+        alert_type: aborted
+        mode: normal
+      put: slack
+    on_error:
+      inputs: []
+      params:
+        alert_type: failed
+        message: Task error, review logs
+        mode: normal
+      put: slack
+    on_failure:
+      inputs: []
+      params:
+        alert_type: failed
+        mode: normal
+      put: slack
+    on_success:
+      inputs: []
+      params:
+        alert_type: success
+        mode: normal
+      put: slack
+    public: true
+    plan:
+      - in_parallel:
+          steps:
+            - get: tagged-release
+              trigger: true
+            - get: android-build-box
+              tags:
+                - large
+              timeout: 5m
+            - do:
+              - task: get-commit-sha
+                config:
+                  platform: linux
+                  image_resource:
+                    type: registry-image
+                    source:
+                      repository: alpine/git
+                  inputs:
+                  - name: tagged-release
+                  outputs:
+                  - name: commit-sha
+
+                  run:
+                    path: sh
+                    args:
+                    - -e
+                    - -x
+                    - -c
+                    - >-
+                        git -C tagged-release rev-parse HEAD > commit-sha/value
+              - load_var: commit
+                file: commit-sha/value
+                format: trim
+                reveal: true
+              - put: internal-test-upload-check
+                inputs: []
+      - task: prepare-env
+        image: android-build-box
+        tags:
+          - large
+        file: tagged-release/ci/pipelines/tasks/prepare-env.yml
+        input_mapping:
+          source: tagged-release
+        output_mapping:
+          source: tagged-release
```

Comment:
If this fails, then the upload check will be stuck as in progress since the only hooks for it are on the other task. I suggest moving the initial `put: internal-test-upload-check` to after this `task`.